griptape/tasks/structure_run_task.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from attr import define, field
+from typing import TYPE_CHECKING
+
+from griptape.artifacts import BaseArtifact, ErrorArtifact
+from griptape.tasks import BaseTextInputTask
+
+if TYPE_CHECKING:
+    from griptape.structures import Structure
+
+
+@define
+class StructureRunTask(BaseTextInputTask):
+    """Task to run a Structure.
+
+    Attributes:
+        structure_to_run: Structure to run.
```

Comment:
Awkward field name but `structure` is already taken by `BaseTask`. Open to suggestions.

Reply:
`target_structure`?

Reply:
Updated